### PR TITLE
feat: tracking for comment modal

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Added tracking for the comment modals 
- 1. comment button
- 2. start discussion button
- added both in companion and webapp/extension
- also removed the popup mode as we don't use it

### On those affected packages:
- [x] Have you done sanity checks in the webapp?
- [x] Have you done sanity checks in the extension?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-126 #done

### CODE REVIEW

File: packages/webapp/pages/posts/[id].tsx [hunk 1]
```diff
@@ -129,7 +129,6 @@ const PostPage = ({ id, postData }: Props): ReactElement => {
         isLoading={isLoading || !isFetched}
         enableAuthorOnboarding={!!router.query?.author}
         enableShowShareNewComment={!!router?.query.new}
-        initializeNewComment={!!router?.query.c}
```
Comment (sshanzel) [inline packages/webapp/pages/posts/[id].tsx:132]: Now that we won't be needing this anymore, especially removed in the `usePostComment`, I think we can also remove the one from the receiving end, which is I think in `PostContent`.

**Review by sshanzel — APPROVED** [review]
Nice. The tracking are well differentiated. Just have one non-blocking comment, but all looks sharp! ✨

File: packages/webapp/pages/posts/[id].tsx (hunk 1 above)
Comment (sshanzel) [reply packages/webapp/pages/posts/[id].tsx:132]: My bad. It was already removed.

**Review by idoshamun — APPROVED** [review]

**Comment by rebelchris** [comment]
Added tracking for the comment modals
comment button
start discussion button
added both in companion and webapp/extension
also removed the popup mode as we don't use it

WT-126 #done

**Pull request merged**